DepotDownloader/Steam3Session.cs

```diff
@@ -112,22 +112,17 @@ public Steam3Session(SteamUser.LogOnDetails details)
             this.callbacks.Subscribe<SteamUser.LoggedOnCallback>(LogOnCallback);
             this.callbacks.Subscribe<SteamUser.SessionTokenCallback>(SessionTokenCallback);
             this.callbacks.Subscribe<SteamApps.LicenseListCallback>(LicenseListCallback);
-            this.callbacks.Subscribe<SteamUser.UpdateMachineAuthCallback>(UpdateMachineAuthCallback);
 
             Console.Write("Connecting to Steam3...");
 
             if (details.Username != null)
             {
-                var fi = new FileInfo(String.Format("{0}.sentryFile", logonDetails.Username));
-                if (AccountSettingsStore.Instance.SentryData != null && AccountSettingsStore.Instance.SentryData.ContainsKey(logonDetails.Username))
+                // Reused what was done for sentryFile but does this has any use ?
```

Comment:
@xPaw can you please view this change?

Reply:
Guard data is used to skip email 2FA. The file stuff can be removed, not sure what the point of it is.